Intersect.Client/Core/Graphics.cs

```diff
@@ -522,9 +522,21 @@ public static void DrawInGame(TimeSpan deltaTime)
         // Draw Entities Overhead Information.
         private static void DrawOverheadInfo(Entity entity)
         {
-            // Who's who.
+            // Cache values.
             var isEvent = entity is Event;
-            bool isNpc = true, isMe = false, isOtherPlayer = false, isFriend = false, isGuildMate = false, isPartyMate = false;
+            bool isNpc = true,
+                isMe = false,
+                isOtherPlayer = false,
+                isFriend = false,
+                isGuildMate = false,
+                isPartyMate = false,
+                myOverheadPref = Globals.Database.MyOverheadInfo,
+                npcOverheadPref = Globals.Database.NpcOverheadInfo,
+                playerOverheadPref = Globals.Database.PlayerOverheadInfo,
+                friendOverheadPref = Globals.Database.FriendOverheadInfo,
+                guildMemberOverheadPref = Globals.Database.GuildMemberOverheadInfo,
+                partyMemberOverheadPref = Globals.Database.PartyMemberOverheadInfo;
```

Comment:
This is just horrible. The code was bad before, this is introducing more lines that only front-load possibly unnecessary computation.